server/src/push/broker.js

```diff
@@ -30,15 +31,21 @@ const sendPush = ({ devices, message }) => {
         console.log('Could not push to device: ', device, e);
       }
     }
-
     if (promises.length === 0) {
       console.log('Tried to push but no valid tokens.');
       return false;
     }
-
     return Promise.all(promises).then(() => true);
   });
 };
 
+const sendTestPush = async ({ devices, message, delay }) => {
+  if (delay) {
+    await sleep(5000);
+    return sendPush({ devices, message });
+  }
+  return sendPush({ devices, message });
```

Comment:
```
return await sendPush({ devices, message });
```

Reply:
bleh.. fine. Just don't forget to add it if you add it if you remove the return and add some additional code afterwards.